### ISSUE TITLE
MIT/BSD license

### ISSUE THREAD

**Issue by glycerine** [unixbigot/gozer#2]
I'd like to build on gozer, but the GPL is a barrier as the code might get used in a commercial setting. Would you be willing to license gozer under a BSD or MIT license?

**Comment by unixbigot** [comment]
Yes I'm happy to relicense as MIT.   I'm out of town at a conference and have limited connection but I'll update when I can.
